tools/cldr-code/src/test/java/org/unicode/cldr/test/TestEn001Time.java

```diff
@@ -0,0 +1,34 @@
+package org.unicode.cldr.test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import org.junit.jupiter.api.Test;
+import org.unicode.cldr.util.*;
+
+public class TestEn001Time {
+
+    static final String TIME_FORMAT_PREFIX =
+            "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/timeFormats/timeFormatLength";
+    static final String TIMEZONE_NAME_PREFIX = "//ldml/dates/timeZoneNames";
+    static final String TIMEZONE_NAME_SHORT = "short";
+
+    /** en_001 should not have standard time formats or any short timezone names */
+    @Test
+    void testTime() {
+        Factory factory = CLDRConfig.getInstance().getCommonAndSeedAndMainAndAnnotationsFactory();
+        CLDRFile cldrFile = factory.make("en_001", false);
+        for (final String path : cldrFile.fullIterable()) {
+            assertFalse(
+                    path.startsWith(TIME_FORMAT_PREFIX),
+                    "Path starts with " + TIME_FORMAT_PREFIX + ": " + path);
+            assertFalse(
+                    path.startsWith(TIMEZONE_NAME_PREFIX) && path.contains(TIMEZONE_NAME_SHORT),
+                    "Path starts with "
+                            + TIMEZONE_NAME_PREFIX
+                            + " + and contains "
+                            + TIMEZONE_NAME_SHORT
+                            + ": "
+                            + path);
```

Comment:
Currently en_001.xml does contain the path "//ldml/dates/timeZoneNames/metazone[@type=\"Hawaii_Aleutian\"]/short/daylight".

https://cldr-smoke.unicode.org/cldr-apps/v#/en_001/NAmerica/39b5dd8bc9bd28b2

Its value is “∅∅∅”.

The ticket description includes, “Check for short timezone names (other than GMT/UTC) and exclude”. Should inclusion of this path with value “∅∅∅” trigger a test failure? Testing locally, this new test does fail on that path.

Reply:
Also note that the test doesn't (yet) treat GMT/UTC differently from other zones

Reply:
As expected, the test fails:
` Error:    TestEn001Time.testTime:24 Path starts with //ldml/dates/timeZoneNames + and contains short: //ldml/dates/timeZoneNames/metazone[@type="Hawaii_Aleutian"]/short/daylight ==> expected: <false> but was: <true>`

Should I make another commit that removes the path in question from en_001.xml? Or, revise the test to allow the value “∅∅∅”?

I just found related documentation here:

https://github.com/unicode-org/cldr/blob/main/docs/site/development/development-process/design-proposals/english-inheritance.md

"Time zone names: en_001 should us “∅∅∅” to cancel the short metazone names inherited from en, for example:"